### PR TITLE
User suggest questions

### PR DESCRIPTION
**Pull request by trungnd95**

### CODE REVIEW

File: app/controllers/questions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class QuestionsController < ApplicationController
+  load_and_authorize_resource
+  skip_authorize_resource only: :index
+  before_action :load_subjects
+
+  def index
+    @questions = Question.where(user_id: current_user.id).page(params[:page])
+      .per(Settings.per_page)
```
Comment (hieunk58) [inline app/controllers/questions_controller.rb]: remove ()

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -5,6 +5,9 @@ def link_to_add_fields name, f, association
     fields = f.fields_for(association, new_object, child_index: id) do |builder|
       render(association.to_s.singularize + "_fields", f: builder)
     end
-    link_to(name, "#", class: "add_fields", data: {id: id, fields: fields.gsub("\n", "")})
+    link_to(name, "#", class: "add_fields btn-floating btn-large
+      waves-effect waves-light red", data: {id: id, fields: fields.gsub("\n", "")})
   end
 end
+
+
```
Comment (hieunk58) [inline app/helpers/application_helper.rb]: remove this line

File: app/models/answer.rb [hunk 3]
```diff
@@ -1,5 +1,4 @@
 class Answer < ActiveRecord::Base
```
Comment (hieunk58) [inline app/models/answer.rb]: checkout this file

File: app/views/questions/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      </div>
+      <div class="row">
+        <div class="input-field col s12">
+          <div class="col m2 center-align">
+            <i class="large material-icons prefix">subject</i>
+          </div>
+          <div class="col m10 push-m2">
+            <%= f.select :subject_id,
+              @subjects.collect {|subject| [subject.name, subject.id ]},
+              {}, {class: "form-control"} %>
```
Comment (hieunk58) [inline app/views/questions/new.html.erb]: indent

File: app/views/questions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        </div>
+      </div>
+      <div id ="answer_field">
+        <div class="col m10 offset-m1">
+          <h5 class="center-align">
+            <%= t "page.admin.questions.answers.answer" %>
+          </h5>
+        </div>
+        <%= f.fields_for :answers do |answer_builder| %>
+        <p>
```
Comment (hieunk58) [inline app/views/questions/new.html.erb]: indent

**Comment by lethuyb** [comment]
OK

**Comment by hieunk58** [comment]
OK

**Pull request merged**